app/views/legal/index.html.erb

```diff
@@ -52,3 +134,6 @@
     </div>
   </div>
 </div>
+
+<%= javascript_pack_tag 'modal' %>
+<%= javascript_pack_tag 'sidebar' %>
```

Comment:
An old practice was to load js files at the bottom of our document, but, tbh, not sure how useful is this

I'd rather move this to the `head` and use named-yields to add them from this view .

For more info: https://guides.rubyonrails.org/layouts_and_rendering.html#using-the-content-for-method